File: test/e2e/testdata/deps/hello_deps.php
<?php
use Cocur\Slugify\Slugify;

function handler(array $input): array {
    $name = $input['name'] ?? 'world';
    $slugify = new Slugify();
    return ['message' => "Hello, {$name}!", 'slug' => $slugify->slugify($name)];
}

File: test/e2e/testdata/deps/hello_deps.rs
use slug::slugify;

fn handler(input: Value) -> Value {
    let name = input.get("name")
        .and_then(|v| v.as_str())
        .unwrap_or("world");
    json!({"message": format!("Hello, {}!", name), "slug": slugify(name)})
}
